api/spec/api_docs/config.rb

```diff
@@ -24,7 +24,7 @@ def swagger_docs
               title: "Manifold",
               version: "v1"
             },
-            host: "localhost:3020", # TODO: base this off the .env file
+            host: ENV["DOMAIN"],
```

Comment:
Environment should not be accessed directly like this. Instead, if you want the URL of the API, get it from here: `Rails.application.config.manifold.api_url`

Reply:
Thanks!